tex/quantum/vectors.tex

```diff
@@ -244,12 +244,15 @@ \section{Observations}
 	\qquad
 	\sigma_y = \begin{bmatrix} 0 & -i \\ i & 0 \end{bmatrix}.
 \]
+
 These matrices are important because:
 \begin{ques}
 	Show that these three matrices, plus the identity matrix,
 	form a basis for the set of Hermitian $2 \times 2$ matrices.
 \end{ques}
-So the Pauli matrices are a natural choice of basis.
+So the Pauli matrices are a natural choice of basis.\footnote{%
+	Well, natural due to physics reasons.
```

Comment:
I think this has something to do with polarization according to SE? not exactly sure.

Reply:
in general i think this will put an extra space after the footnote in the bottom, so we want to do something more like `\footnote{Well,` and break the line after the first word